Makefile

```diff
@@ -97,9 +97,9 @@ $(KUBECTL_BIN): $(SHELL_ENV_EXISTS)
 	$(V)mv $@.tmp $@
 
 KUBECTL = $(VENV_BIN)/kubectl
-KUBECTL_REALPATH = realpath $(KUBECTL_BIN)
+KUBECTL_REALPATH = $(PWD)/$(KUBECTL_BIN)
 $(KUBECTL): $(KUBECTL_BIN) $(VENV_EXISTS)
-	$(V)ln -sf $(shell $(KUBECTL_REALPATH))$(KUBECTL_REALPATH:sh) $@
+	$(V)ln -sf $(KUBECTL_REALPATH)$(KUBECTL_REALPATH:sh) $@
```

Comment:
Now, this is no longer POSIX `make` compliant now...

Reply:
Oups sorry, don't know every bit of POSIX. Which part is not posix ?

Reply:
Ah indeed, this will break with setting SHELL_ENV (Can we just make a distinction between shell_env starting with / or not ?). We can also remove this part, and require realpath (which seem to be a C POSIX function and not a command). The proper fix of the ticket is the backquotes bellow